src/JuliusSweetland.OptiKey.Core/Services/Translation/TranslationService.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) 2019 OPTIKEY LTD (UK company number 11854839) - All Rights Reserved
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Diagnostics;
+
+using System.Net.Http;
+using System.IO;
+using Newtonsoft.Json;
+using System.CodeDom.Compiler;
+using System.CodeDom;
+using JuliusSweetland.OptiKey.Properties;
+
+namespace JuliusSweetland.OptiKey.Services.Translation
+{
+    public class TranslationService
+    {
+        private HttpClient client;
+        private string apiKeyToUse;
+        public const string LICENSE_TEXT = "(Powered by Yandex.Translate)";
+
+        public TranslationService(HttpClient httpClient)
+        {
+            this.client = httpClient;
+        }
+
+        public struct Response
+        {
+            public string status;
```

Comment:
Could these be recased to be Title Cased please?

Also I wouldn't mind a constructor - structs don't NEED to be newed up, but if you don't call the constructor (and set the properties via the constructor) then you need to set ALL of the properties before the struct can be used. I just prefer the unambiguous approach of using a ctor.